fastfilter/src/main/java/org/fastfilter/bloom/BlockedBloom.java

```diff
@@ -34,7 +34,7 @@ public long getBitCount() {
         this.seed = Hash.randomSeed();
         long bits = (long) entryCount * bitsPerKey;
         this.buckets = (int) bits / 64;
-        data = new long[(int) (buckets + 16)];
+        data = new long[(buckets + 16)];
```

Comment:
The array is not large enough. The right fix for the ArrayOutOfBounds is to increase the array size: 1 larger for the "+ 1", and then 15 larger for the "hash >>> 60". So I think 16 entries larger.